tests/transformers/test_transformers_model_export.py

```diff
@@ -3882,3 +3885,33 @@ def test_log_model_skip_validating_serving_input_for_local_checkpoint(
         mock_validate_input.assert_not_called()
     else:
         mock_validate_input.assert_called_once()
+
+
+def test_log_model_sends_telemetry_record(mock_requests, base_model):
```

Comment:
do we need to test all flavors? Can we pick one and test it in `tests/telemetry`?

Reply:
Based on my previous tests, it's easy to miss edge cases :( The test itself is not complex so I just add for all flavors

Reply:
Got it, let's keep them :)